lib/generator/inputLatencyFrameThroughputGenerator.py

```diff
@@ -0,0 +1,277 @@
+import os
+import json
+import time
+import copy
+import numpy as np
+from baseGenerator import BaseGenerator
+from frameThroughputDctGenerator import FrameThroughputDctGenerator
+from ..helper.terminalHelper import find_terminal_view
+from ..common.visualmetricsWrapper import find_tab_view
+from ..common.visualmetricsWrapper import find_image_viewport
+from ..common.imageUtil import generate_crop_data
+from ..common.imageUtil import crop_images
+from ..common.imageUtil import convert_to_dct
+from ..common.imageUtil import find_browser_view
+from ..common.imageUtil import compare_with_sample_image_multi_process
+from ..common.imageUtil import CropRegion
+from ..common.commonUtil import CommonUtil
+from ..common.logConfig import get_logger
+
+logger = get_logger(__name__)
+
+
+class CombinationGenerator(BaseGenerator):
+
+    # We should not skip fraction when it comes to terminal region
+    FULL_FRACTION = 1.0
+    # It is used only when you want to get rid of status bar or panel on bottom left
+    SKIP_STATUS_BAR_FRACTION = 0.95
+
+    BROWSER_VISUAL_EVENT_POINTS = {
+        # (IL start) <--- time ---> ( FT Start = IL end )
+        'backward_search': [
+            {'event': 'il_end', 'search_target': CropRegion.VIEWPORT, 'fraction': SKIP_STATUS_BAR_FRACTION, 'shift_result': True},
+            {'event': 'il_start', 'search_target': CropRegion.TERMINAL, 'fraction': FULL_FRACTION, 'shift_result': True}
+        ],
+        # ( FT Start = IL end ) <--- time ---> (FT end)
+        'forward_search': [
+            {'event': 'ft_end', 'search_target': CropRegion.VIEWPORT, 'fraction': SKIP_STATUS_BAR_FRACTION}
```

Comment:
The `start` and `end` are special events.
Modifying them might effects other classes and methods.

I will suggest to add some message on your commit's comment.